### PR TITLE
sanbox ui fixes

### PR DESCRIPTION
**Pull request by SidHarsh25**

### CODE REVIEW

**Comment by SerhiiStarovoitov** [comment]
@ccfredbaum, this one contains CSS edits from Sid. Can be merged to main.

**Review by SerhiiStarovoitov — APPROVED** [review]

File: elemental/module/membership/library/class-loginhandler.php [hunk 1]
```diff
@@ -33,6 +33,9 @@ class LoginHandler {
 	const SHORTCODE_LOGIN_BUTTON   = 'elemental_loginbutton';
 	const SHORTCODE_LOGIN_REDIRECT = 'elemental_profile_redirect';
 
+	const SHORTCODE_COMPANY_EDIT = 'elemental_compant_edit';
```
Comment (ccfredbaum) [inline elemental/module/membership/library/class-loginhandler.php:36]: Should this be company_edit

File: elemental/module/membership/library/class-loginhandler.php [hunk 2]
```diff
@@ -60,7 +63,28 @@ public function init() {
...
 	}
+
+	/**
+	 * Initialise LoginHandler Ajax.
+	 */
+	public function initialise_loginhandler_ajax(): void
+	{
+
+		\add_action('wp_ajax_elemental_editcompany_ajax', array(Factory::get_instance(CompanyAjax::class), 'company_ajax_handler'), 10, 2);
+		// \add_action('wp_ajax_nopriv_send_contact',array(Factory::get_instance(LoginAjaxHandler::class), 'smtp_ajax_handler'), 10, 2);
```
Comment (ccfredbaum) [inline elemental/module/membership/library/class-loginhandler.php:83]: Please remove commented out actions.

File: elemental/module/membership/library/class-loginhandler.php [hunk 3]
```diff
@@ -440,4 +464,79 @@ public function loginland_redirect() {
 			return Factory::get_instance( ElementalBP::class )->bp_profile_redirect();
 		}
 	}
+
+
+	//Registering Script
```
Comment (ccfredbaum) [inline elemental/module/membership/library/class-loginhandler.php:469]: Doc Block comments. Please make them consistent to main including vars, outputs and params

**Review by ccfredbaum — CHANGES_REQUESTED** [review]

File: elemental/module/membership/library/class-loginhandler.php (hunk 3 above)
Comment (SidHarsh25) [reply elemental/module/membership/library/class-loginhandler.php]: Documented all block comments.

File: elemental/module/membership/library/class-loginhandler.php (hunk 2 above)
Comment (SidHarsh25) [reply elemental/module/membership/library/class-loginhandler.php]: removed unwanted and commented methods

File: elemental/module/membership/library/class-loginhandler.php (hunk 1 above)
Comment (SidHarsh25) [reply elemental/module/membership/library/class-loginhandler.php]: changed and to elemental_company_edit and pushed

File: elemental/module/membership/library/class-companyajax.php [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                    $user = get_user_by('id', $result);
+                    //handle successful creation here
+                     $response['feedback'] = \esc_html__('User Created', 'elementalplugin');
+                    }
+            }
+        
+        }
+
+        //    public function randomPassword() {
+        //     $alphabet = "abcdefghijklmnopqrs)-(@&!tuwxyzABCDEFGHIJKLM)-(@&!tNOPQRSTUWXYZ01234)-(@&!t56789";
```
[the remark this answers is not in the document]
Comment (SidHarsh25) [reply elemental/module/membership/library/class-companyajax.php]: removed this method

**Pull request merged**